gui/src/components/preboarding/admin-user.vue

```diff
@@ -248,9 +249,13 @@ import {
 } from "./../../common/regex-helpers";
 import { invalid } from "moment";
 import i18n from "./../../i18n";

```

Comment:
Please add component translation instead of **i18n**

Reply:
in cortx1.0-dev branch we are not adding those changes